server/index.js

```diff
@@ -3,13 +3,16 @@ const app = express();
 const port = 5000;
 const queries = require('./../db/queries');
 
+app.use(express.static('./public'))
+app.use(express.static('./client/dist'))
 
```

Comment:
Is there a reason for using two of these- app.us(express.static())? I know it's used for loading the html file, but do you have a client/dist folder? I didn't see one...

Reply:
yeah im not too sure about where to place them. i asked leslie about folder structure, and well, it seemed kinda gray. for sure she said that public is for static assets like pictures, and i guess static html...but yeah, it wasn't very clear.

but yeah, good idea. i think i might try putting it all in the public folder so i only have to serve up one.
possibly:

public
  -dist
    -bundle.js
  -index.html

the css i might just leave it in the src since it gets compiled along with everything else.

and for the .gitignore, they had the dist (or build) keywords in the file. so i just went with that.

Reply:
*
public
...dist
.........bundle.js
...index.html